services/housetables/src/main/java/com/linkedin/openhouse/housetables/repository/impl/jdbc/UserTableHtsJdbcRepository.java

```diff
@@ -64,14 +65,16 @@ Page<UserTableRow> findAllByDatabaseIdAndTableIdLikeAllIgnoreCase(
           + "(:tableVersion IS NULL OR u.version = :tableVersion) AND "
           + "(:metadataLocation IS NULL OR u.metadataLocation = :metadataLocation) AND "
           + "(:storageType IS NULL OR u.storageType = :storageType) AND "
-          + "(:creationTime IS NULL OR u.creationTime = :creationTime)")
+          + "(:creationTime IS NULL OR u.creationTime = :creationTime) AND "
+          + "(:deleted IS FALSE OR u.deleted = :deleted)")
```

Comment:
This method seems to be filtering based on the deleted flag.

Reply:
Ah sorry I fixed this in the paginated method but not the regular one, fixed